drivers/pppos.c

```diff
@@ -738,6 +745,8 @@ static int pppos_netifInit(struct netif *netif, char *cfg)
 		}
 		netif->flags &= ~NETIF_FLAG_UP;
 		ppp_set_netif_statuscallback(state->ppp, pppos_statusCallback);
+		if (!(flags & CFG_FLAG_NO_DEFAULT_ROUTE))
+			ppp_set_default(state->ppp);
```

Comment:
for single-interface targets it doesn't matter, but be aware that it might not work with our `route` module as intended (maybe it should (the `#if 0` code paths), I haven't checked it further)

Reply:
So maybe, let's reverse the logic for safety? So the user should provide `defaultroute` argument instead of `nodefaultroute`. It would be different than `pppou.c`, but we can change it there too.

Reply:
we should probably use our routing api instead, but I think the implementation is messy (with some logic commented out) so let's leave it as-is for now (straightening how `route` works should be a topic of separate PR)